src/h5web/toolbar/OverflowMenu.tsx

```diff
@@ -15,15 +14,11 @@ function OverflowMenu(props: Props) {
   const rootRef = useRef(null);
   const [isOverflowMenuOpen, toggleOverflowMenu] = useToggle(false);
 
-  useClickAway(
-    rootRef,
-    () => {
-      if (isOverflowMenuOpen) {
-        toggleOverflowMenu(false); // force close in case menu button was clicked
```

Comment:
The double state update when clicking the button doesn't occur any more since I had previously moved the button inside the `rootRef` element.